### PR TITLE
Changes for bridge port tagging mode

### PR DESCRIPTION
**Pull request by dgsudharsan**
Changes to make a bridge port tagged or untagged

### CODE REVIEW

**Comment by lguohan** [comment]
retest this please

File: inc/saibridge.h [hunk 1]
```diff
@@ -88,6 +88,19 @@ typedef enum _sai_bridge_port_type_t
...
+ */
+typedef enum _sai_bridge_port_tagging_mode_t
+{
+    /** Untagged mode */
+    SAI_BRIDGE_PORT_TAGGING_MODE_UNTAGGED,
+
+    /** Tagged mode */
+    SAI_BRIDGE_PORT_TAGGING_MODE_TAGGED,
+
+}sai_bridge_port_tagging_mode_t;
```
Comment (kcudnik) [inline inc/saibridge.h]: missing space after }

File: inc/saibridge.h [hunk 2]
```diff
@@ -118,12 +131,21 @@ typedef enum _sai_bridge_port_attr_t
...
+    SAI_BRIDGE_PORT_ATTR_TAGGING_MODE,
+
+    /**
      * @brief Associated Vlan
      *
      * @type sai_uint16_t
      * @flags MANDATORY_ON_CREATE | CREATE_ONLY
      * @isvlan true
-     * @condition SAI_BRIDGE_PORT_ATTR_TYPE == SAI_BRIDGE_PORT_TYPE_SUB_PORT
+     * @condition SAI_BRIDGE_PORT_ATTR_TAGGING_MODE == SAI_BRIDGE_PORT_TAGGING_MODE_TAGGED
```
Comment (kcudnik) [inline inc/saibridge.h]: also one conditional attribute depends on another conditional attribute, this is not supported now

**Comment by lguohan** [comment]
still need to meet the requirement to match untagged packet and assigned to the bridge port.

**Pull request merged**